invenio_search/config.py

```diff
@@ -59,4 +56,4 @@
     "980__b": ["collections.secondary"],
     "542__l": ["information_relating_to_copyright_status.copyright_status"],
 }
-"""Holds a dictionary to map invenio keywords to elasticsearch fields."""
+"""Holds a dictionary to map Invenio keywords to Elasticsearch fields."""
```

Comment:
It might be we can also remove this mapping (``SEARCH_ELASTIC_KEYWORD_MAPPING``) - can you check?

Reply:
`SEARCH_ELASTIC_KEYWORD_MAPPING` is used in the `elasticsearch.py` walker: https://github.com/inveniosoftware/invenio-search/blob/master/invenio_search/walkers/elasticsearch.py#L33 so I left it.
On the other hand I don't see anyone actually using that `ElasticSearchDSL` class at all (none of the modules, nor Zenodo, CDS, B2Share or Inspire), so maybe we can remove the file completely? If we remove it, maybe we can remove the `invenio-query-parser` from `invenio-search` (I remember there was a reason why we had it there, but I can't recall why, and I see that this file has the only reference to `invenio-query-parser` in the whole module) ?

Reply:
I would be good to remove the dependency on invenio-query-parser if possible. Probably we can just move ``invenio_search.walkers`` to invenio-query-parser (to make it for tomorrow I think we can remove it from invenio-search, and add a ticket to invenio-query-parser to add the walker in the next sprint).

Reply:
Ok, I removed the walker directory completely. I will add the issue to the `invenio-query-parser` tomorrow (I'm fighting with the documentation now) - unless you want to describe what exactly needs to be done (I see that `invenio-query-parser` already has a `ElasticSearchDSL` class in the walkers directory, but it looks different that this one, so there might be more to do than just copy the removed code there)?